dl/src/main/scala/com/intel/analytics/bigdl/nn/BiRecurrent.scala

```diff
@@ -25,18 +25,15 @@ import com.intel.analytics.bigdl.utils.T
 import scala.reflect.ClassTag
 
 class BiRecurrent[T : ClassTag] (
-  layer: Recurrent[T], revLayer: Recurrent[T])
+  layer: Recurrent[T], revLayer: Recurrent[T], timeDim: Int = 2)
```

Comment:
The default timeDim is 2 in Recurrent. Shall we remove this parameter to set default timeDim?